pliers/extractors/audio.py

```diff
@@ -618,3 +634,67 @@ def _extract(self, stim):
         return ExtractorResult(preds, stim, self, features=labels,
                                onsets=onsets, durations=[dur]*len(onsets),
                                orders=list(range(len(onsets))))
+
+class MFCCLowEnergyExtractor(MFCCExtractor):
+    ''' Given a Middle Bound, Extract the Energy from the Low Register
+    of Mel-Frequency Ceptral Coefficients.
+
+    Args:
+    n_mfcc (int): specifies the number of MFCC to extract
+    n_coefs(int): cepstrum coefficients to keep in the low quefrency spectrum
+    hop_length (int): hop length in number of samples
+    librosa_kwargs (optional): Optional named arguments to pass to librosa
+    '''
+
+    _log_attributes = ('hop_length', 'n_coefs', 'n_mfcc')
+
+    def __init__(self, n_mfcc=48, n_coefs=13, hop_length=1024, **librosa_kwargs):
+        self.n_mfcc = n_mfcc
+        self.n_coefs = n_coefs
+        self.hop_length = hop_length
+        self.librosa_kwargs = librosa_kwargs
+        super().__init__(n_mfcc=n_mfcc)
+
+    @override
+    def _proc_func(self,stim):
+        vals = self._get_values(stim)
+        low_mfs = fft.dct(np.transpose(vals[:self.n_coefs]), 
+            type=2, 
+            n=self.n_mfcc, 
+            axis=- 1, 
+            norm='ortho', 
+            overwrite_x=False)
+        low_mfs = 10 ** (low_mfs / 20.)
+        return low_mfs.T
+
+class MFCCHighEnergyExtractor(MFCCExtractor):
+    ''' Given a Middle Bound, Extract the Energy from the High Register
+    of Mel-Frequency Ceptral Coefficients.
+
+    Args:
+    n_mfcc (int): specifies the number of MFCC to extract
+    n_coefs(int): cepstrum coefficients to keep in the high quefrency spectrum
+    hop_length (int): hop length in number of samples
+    librosa_kwargs (optional): Optional named arguments to pass to librosa
+    '''
+
+    _log_attributes = ('hop_length', 'n_coefs', 'n_mfcc')
+
+    def __init__(self, n_mfcc=48, n_coefs=35, hop_length=1024, **librosa_kwargs):
+        self.n_mfcc = n_mfcc
+        self.n_coefs = n_coefs
+        self.hop_length = hop_length
+        self.librosa_kwargs = librosa_kwargs
+        super().__init__(n_mfcc=n_mfcc)
+
+    @override
+    def _proc_func(self,stim):
```

Comment:
I see where your going with this, but I'd like to avoid using "custom" ways of over-riding functionality, especially the `override` decorator (since this is not "standard" python). The main reason I say this is because this will reduce maintenance burdens in the future since it will be more obvious what this is doing, if it is done in a more "standard" way.

I think you should be able to accomplish this same behavior using `super()`. It seems that  you should be able to override the `_get_values` function with the contact of this `_proc_func`. Instead of calling `self._get_values` you would call `super()._get_values` to get the MFCCs, then post-processes them further and return the energy extracted from the MFCCs.

I haven't thought it through completely but I believe this should let you do what you're doing here, without manually keeping track of whether a function as been over-ridden or not. `_get_values` should simply return the values of whatever the current extractor is.